doc/release/1.11.0-notes.rst

```diff
@@ -1,14 +1,25 @@
 NumPy 1.11.0 Release Notes
 **************************
 
-This release supports Python 2.6 - 2.7 and 3.2 - 3.5.
+This release supports Python 2.6 - 2.7 and 3.2 - 3.5 and contains a number
+of enhancements and improvements. Note also the build system changes listed
+below as they may have subtle effects.
+
+No Windows (TM) binaries are provided for this release due to a broken
+toolchain. One of the providers of Python packages for Windows (TM) is your
+best bet.
 
 
 Highlights
 ==========
 
-* The datetime64 type is now timezone naive. See "datetime64 changes" below
-  for more details.
+Details of these improvements can be found below.
+
+* The datetime64 type is now timezone naive.
+* A dtype parameter has been added to ``randint``.
+* A new benchmark suite using `Airspeed Velocity`.
```

Comment:
The benchmark suite isn't user facing, so I would probably drop it as a highlight for the release notes.

FWIW, my favorite changes for the release (beyond your list) include:
- automatic bin width estimation for `np.histogram`
- `np.moveaxis` (full disclosure: yes, this was mine!)
- optimization for `A @ A.T`